Functions.Templates/PortalTemplates.nuspec

```diff
@@ -85,13 +85,24 @@
     <file src="templates/DurableFunctionsActivity-CSharp/DurableFunctionsActivityCSharp.csx" target="templates/DurableFunctionsActivity-CSharp/run.csx" />
     <file src="templates/DurableFunctionsActivity-CSharp/sample.dat" target="templates/DurableFunctionsActivity-CSharp/sample.dat" />
+    <file src="templates/DurableFunctionsActivity-JavaScript/function.json" target="Templates/DurableFunctionsActivity-JavaScript/function.json" />
+    <file src="templates/DurableFunctionsActivity-JavaScript/index.js" target="templates/DurableFunctionsActivity-JavaScript/index.js" />
+    <file src="templates/DurableFunctionsActivity-JavaScript/metadata.json" target="Templates/DurableFunctionsActivity-JavaScript/metadata.json" />
+    <file src="templates/DurableFunctionsActivity-JavaScript/sample.dat" target="Templates/DurableFunctionsActivity-JavaScript/sample.dat" />    
     <file src="templates/DurableFunctionsOrchestrator-CSharp/function.json" target="templates/DurableFunctionsOrchestrator-CSharp/function.json" />
     <file src="templates/DurableFunctionsOrchestrator-CSharp/metadata.json" target="templates/DurableFunctionsOrchestrator-CSharp/metadata.json" />
     <file src="templates/DurableFunctionsOrchestrator-CSharp/DurableFunctionsOrchestratorCSharp.csx" target="templates/DurableFunctionsOrchestrator-CSharp/run.csx" />
     <file src="templates/DurableFunctionsOrchestrator-CSharp/sample.dat" target="templates/DurableFunctionsOrchestrator-CSharp/sample.dat" />
+    <file src="templates/DurableFunctionsOrchestrator-JavaScript/function.json" target="Templates/DurableFunctionsOrchestrator-JavaScript/function.json" />
+    <file src="templates/DurableFunctionsOrchestrator-JavaScript/index.js" target="templates/DurableFunctionsOrchestrator-JavaScript/index.js" />
+    <file src="templates/DurableFunctionsOrchestrator-JavaScript/metadata.json" target="Templates/DurableFunctionsOrchestrator-JavaScript/metadata.json" />
+    <file src="templates/DurableFunctionsOrchestrator-JavaScript/sample.dat" target="Templates/DurableFunctionsOrchestrator-JavaScript/sample.dat" />    
     <file src="templates/DurableFunctionsHttpStart-CSharp/function.json" target="templates/DurableFunctionsHttpStart-CSharp/function.json" />
     <file src="templates/DurableFunctionsHttpStart-CSharp/metadata.json" target="templates/DurableFunctionsHttpStart-CSharp/metadata.json" />
     <file src="templates/DurableFunctionsHttpStart-CSharp/DurableFunctionsHttpStartCSharp.csx" target="templates/DurableFunctionsHttpStart-CSharp/run.csx" />
+    <file src="templates/DurableFunctionsHttpStart-JavaScript/function.json" target="templates/DurableFunctionsHttpStart-JavaScript/function.json" />
+    <file src="templates/DurableFunctionsHttpStart-JavaScript/metadata.json" target="templates/DurableFunctionsHttpStart-JavaScript/metadata.json" />
+    <file src="templates/DurableFunctionsHttpStart-JavaScript/index.js" target="templates/DurableFunctionsHttpStart-JavaScript/index.js" />    

```

Comment:
no sample for DurableFunctionsHttpStart-JavaScript?

Reply:
The default orchestration scenario in DurableFunctionsOrchestrator-JavaScript doesn't take any input to run, so no.